stock_quant_editable_view_block/hooks.py

```diff
@@ -0,0 +1,56 @@
+# Copyright 2018-20 ForgeFlow <http://www.forgeflow.com>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/lgpl.html).
+from odoo import _, api
+
+from odoo.addons.stock.models.stock_quant import StockQuant
+
+
+def post_load_hook():
+    @api.model
+    def _new_get_quants_action(self, domain=None, extend=False):
```

Comment:
I'm not really in favor of such things as you lose all the inheritance mechanism.

Isn't there any other mean doing the same thing ?

Reply:
Create a new action and overwrite on views. What is preferable?

Reply:
And in this case, why not overriding original method directly ???

Reply:
At this point, it is a matter of design, or am I missing anything?

This way it will also work! :smiley_cat: 
```
# Copyright 2018-20 ForgeFlow <http://www.forgeflow.com>

from odoo import api, models


class StockQuant(models.Model):
    _inherit = 'stock.quant'

    @api.model
    def _get_quants_action(self, domain=None, extend=False):
        action = super()._get_quants_action(domain=domain, extend=extend)
        action['view_id'] = self.env.ref('stock.view_stock_quant_tree').id
        # Enables form view in readonly list
        action.update({
            'view_mode': 'tree,form',
            'views': [
                (action['view_id'], 'list'),
                (self.env.ref('stock.view_stock_quant_form').id, 'form'),
            ],
        })
        return action
```

Reply:
I changed the override for that code again, it should perform the same, could we move it forward?